plugins/action/openssl_privatekey_pipe.py

```diff
@@ -0,0 +1,110 @@
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Felix Fontein <felix@fontein.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+import base64
+
+from ansible.module_utils._text import to_native, to_bytes
+
+from ansible_collections.community.crypto.plugins.module_utils.action_module import ActionModuleBase
```

Comment:
If I understand it correctly, this helper stuff is part of Ansible 2.11 already - maybe we could make this import conditional to Ansible <=2.10?

Reply:
ActionModuleBase does not exist like this in any ansible-base version to my knowledge.

Reply:
:crying_cat_face: 

Let's hope something like it eventually ends up upstream - action plugins are unnecessarily hard to write in some cases.

Reply:
I hope so too. I know that more of the argument spec handling will get extracted for ansible-base 2.11, since there will be an action plugin for validating role argument specs. It will be interesting to see how that looks in the end, and how does it help with this task.